### ISSUE TITLE
Upgraded to 7.2.0, Now I'm getting "Oops something went wrong" when I open PowerShell 7

### ISSUE THREAD

**Issue by dclajoie** [PowerShell/PSReadLine#3003]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
PS C:\Program Files\PowerShell\7>
Oops, something went wrong.
Please report this bug with ALL the details below, including both the 'Environment' and 'Exception' sections.
Please report on GitHub: https://github.com/PowerShell/PSReadLine/issues/new?template=Bug_Report.md
Thank you!

### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.22000
BufferWidth: 120
BufferHeight: 9001

Last 0 Keys


```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```
```


### Screenshot

![Screenshot 2021-11-10 092909](https://user-images.githubusercontent.com/46324931/141142172-237634cb-7cf5-4771-bf89-3657819079c8.png)


### Environment data

```console
PS Version: 7.2.0
PS HostName: ConsoleHost
PSReadLine Version:
PSReadLine EditMode: Windows
OS: 10.0.22000.1 (WinBuild.160101.0800)
BufferWidth: 120
BufferHeight: 9001
```

### Steps to reproduce

I just open PowerShell 7.2 and it happens.

### Expected behavior

Should open correctly.

### Actual behavior

The error above occurs.

**Comment by dc-chris-smith** [comment]
I am experiencing this same issue with the same results presented.

**Comment by valuecoder** [comment]
Also started seeing this behavior yesterday with same version of PoSh running on Windows 11

**Comment by ajhanson** [comment]
experiencing same issue on Win10 after upgrade to 7.2

**Comment by dc-chris-smith** [comment]
I even ran the following command from WPS to make sure it was the latest release version of PSReadLine

```
pwsh -command {Install-Module -Name PSReadLine -RequiredVersion 2.1.0 -Force}
```
The response was
```
WARNING: The version '2.1.0' of module 'PSReadLine' is currently in use. Retry the operation after closing the applications.
```

**Comment by nicknguyen** [comment]
> I even ran the following command from WPS to make sure it was the latest release version of PSReadLine
> 
> ```
> pwsh -command {Install-Module -Name PSReadLine -RequiredVersion 2.1.0 -Force}
> ```
> 
> The response was
> 
> ```
> WARNING: The version '2.1.0' of module 'PSReadLine' is currently in use. Retry the operation after closing the applications.
> ```

This worked for me despite the command completing w/ an admin rights error even though running as an admin:
`pwsh -command {Install-Module -Name PSReadLine -RequiredVersion **2.2.0-beta4** -Force -AllowPrerelease -Scope CurrentUser }`

Note the version 2.2.0-beta4

**Comment by dqfan2012** [comment]
I'm getting the same error. Is there a way to rollback to the previous version?

**Comment by iSazonov** [comment]
Please update to latest Beta4 version.

**Comment by daxian-dbw** [comment]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.
